### DISCUSSION TITLE
How I can hide the menubar?

### DISCUSSION THREAD

**Discussion by xmha97** [hluk/CopyQ#2033] [Q&A]
How I can hide the menubar?

![CopyQ](https://user-images.githubusercontent.com/23485114/177266022-2851f775-19c1-47eb-b080-77f3e9d489a8.PNG)

**Comment by hluk** [comment]
See: [FAQ: How to hide the menu bar in the main CopyQ window?](https://copyq.readthedocs.io/en/latest/faq.html#how-to-hide-the-menu-bar-in-the-main-copyq-window)
